<ui version="4.0" >
 <class>FormAudioConversor</class>
 <widget class="QWidget" name="FormAudioConversor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>AudioConversor</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/small_logo.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/small_logo.png</iconset>
  </property>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap>../../../../../usr/share/kademar/utils/cadi/img/bg/cadi.png</pixmap>
   </property>
   <property name="scaledContents" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="pages" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>40</y>
     <width>591</width>
     <height>541</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="files" >
    <widget class="QPushButton" name="b_chooseFormat" >
     <property name="geometry" >
      <rect>
       <x>320</x>
       <y>460</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Choose Format</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/next.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/next.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>200</x>
       <y>99</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text" >
      <string>Convert all your Song files and Extract the audio tracks from your Favourite Videos.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>180</y>
       <width>471</width>
       <height>231</height>
      </rect>
     </property>
     <property name="acceptDrops" >
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_10" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>10</y>
       <width>291</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap>../../../../../usr/share/kademar/utils/audioconversor/img/logo.png</pixmap>
     </property>
    </widget>
    <zorder>label_10</zorder>
    <zorder>b_chooseFormat</zorder>
    <zorder>label_4</zorder>
    <zorder>listWidget</zorder>
   </widget>
   <widget class="QWidget" name="convert" >
    <widget class="QPushButton" name="b_back" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>460</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text" >
      <string>Back</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/back.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/back.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_final" >
     <property name="geometry" >
      <rect>
       <x>320</x>
       <y>460</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Convert Audio</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/media_podcast.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/media_podcast.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_format" >
     <property name="geometry" >
      <rect>
       <x>230</x>
       <y>138</y>
       <width>79</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text" >
       <string>MP3</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>OGG</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>WAV</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>140</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Convert to</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="l_bitrate" >
     <property name="geometry" >
      <rect>
       <x>60</x>
       <y>190</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Resolution/Bitrate</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_bitrate" >
     <property name="geometry" >
      <rect>
       <x>230</x>
       <y>188</y>
       <width>79</width>
       <height>25</height>
      </rect>
     </property>
     <property name="currentIndex" >
      <number>7</number>
     </property>
     <item>
      <property name="text" >
       <string>32</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>40</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>48</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>56</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>64</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>80</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>96</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>112</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>128</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>160</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>192</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>224</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>256</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>320</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit" >
     <property name="geometry" >
      <rect>
       <x>110</x>
       <y>310</y>
       <width>261</width>
       <height>26</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>280</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Put converted files to Folder:</string>
     </property>
    </widget>
    <widget class="QToolButton" name="b_browse" >
     <property name="geometry" >
      <rect>
       <x>400</x>
       <y>310</y>
       <width>30</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/folder.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/folder.png</iconset>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider" >
     <property name="geometry" >
      <rect>
       <x>230</x>
       <y>193</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>5</number>
     </property>
     <property name="singleStep" >
      <number>1</number>
     </property>
     <property name="pageStep" >
      <number>5</number>
     </property>
     <property name="value" >
      <number>3</number>
     </property>
     <property name="tracking" >
      <bool>true</bool>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QLabel" name="l_num_bitrate" >
     <property name="geometry" >
      <rect>
       <x>410</x>
       <y>190</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>40</y>
       <width>461</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Select your Format</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page" >
    <widget class="QListWidget" name="listWidget_2" >
     <property name="geometry" >
      <rect>
       <x>50</x>
       <y>140</y>
       <width>481</width>
       <height>271</height>
      </rect>
     </property>
     <property name="acceptDrops" >
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>170</x>
       <y>40</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font" >
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Convert Process</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>90</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>This process can take a while. Please be patient</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_stop" >
     <property name="geometry" >
      <rect>
       <x>320</x>
       <y>460</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" >
      <string>Â¡Stop Process!</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/process_stop.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/process_stop.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_back_2" >
     <property name="geometry" >
      <rect>
       <x>70</x>
       <y>460</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text" >
      <string>Back</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/back.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/back.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="b_sortir_2" >
     <property name="geometry" >
      <rect>
       <x>320</x>
       <y>460</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" >
      <string>Exit</string>
     </property>
     <property name="icon" >
      <iconset>
       <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/exit.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/exit.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>10</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="pixmap" >
    <pixmap>../../../../../usr/share/kademar/utils/cadi/img/kademar.png</pixmap>
   </property>
   <property name="scaledContents" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="b_sortir" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>530</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/exit.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/exit.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="b_addFile" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>190</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add File</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/add.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/add.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="autoExclusive" >
    <bool>false</bool>
   </property>
   <property name="autoDefault" >
    <bool>false</bool>
   </property>
   <property name="default" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="b_addFolder" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>240</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add Folder</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/add_folder.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/add_folder.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="b_removeFile" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>320</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Remove File</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/remove.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/remove.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="b_clearList" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>380</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Clear List</string>
   </property>
   <property name="icon" >
    <iconset>
     <normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/list_clean.png</normaloff>../../../../../usr/share/kademar/utils/audioconversor/img/list_clean.png</iconset>
   </property>
   <property name="iconSize" >
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
